test/python/pythoncall.jl

```diff
@@ -1,10 +1,12 @@
-# PyCall and PythonCall must use the same Python interpreter. This environment variable
-# tells PythonCall to use the same Python interpreter as PyCall. See
-# https://github.com/JuliaPy/PythonCall.jl/blob/5f56a9b96b867a9f6742ab1d1e2361abd844e19f/docs/src/pycall.md#tips
-ENV["JULIA_PYTHONCALL_EXE"] = "@PyCall"
-
 using DifferentialEquations, PythonCall
 
+py""" # This is a mess because normal site-packages is not writeable in CI
+import subprocess, sys, site
+subprocess.run([sys.executable, '-m', 'pip', 'install', '--user', 'julia'])
+subprocess.run([sys.executable, '-m', 'pip', 'install', '--user', 'diffeqpy'])
+sys.path.append(site.getusersitepackages())
+"""
```

Comment:
```suggestion
pyexec(""" # This is a mess because normal site-packages is not writeable in CI
import subprocess, sys, site
subprocess.run([sys.executable, '-m', 'pip', 'install', '--user', 'julia'])
subprocess.run([sys.executable, '-m', 'pip', 'install', '--user', 'diffeqpy'])
sys.path.append(site.getusersitepackages())
""")
```